src/cli/build.ts

```diff
@@ -82,6 +83,9 @@ export default function (program: RootCmd): void {
       ),
     )
     .action(async opts => {
+      // Validate build arguments
+      validateBuildArgs(opts);
```

Comment:
Actually... can we just use [Commander.js .conflicts() options](https://github.com/tj/commander.js/blob/master/examples/options-conflicts.js) to handler tracking whether bad option combos are in use?  Or is there a reason we had to try to handle it ourselves withing the command body?